firmware/application/apps/ui_mictx.cpp

```diff
@@ -375,9 +375,9 @@ MicTXView::MicTXView(
                 rxaudio(rx_enabled);         // Update now if we have RX audio on
                 options_tone_key.hidden(0);  // we are in WFM mode, we should have active the Key-tones & CTCSS option.
 
-                rxbw.emplace_back("  200k-WFM ", 0);  // We allow the user selection of the 3 x WFM BW filters, (0) WFM-200K, (1) WFM-180K, (2) WFM-40K.
-                rxbw.emplace_back("  180k-WFM ", 1);
-                rxbw.emplace_back("   40k-WFM ", 2);
+                rxbw.emplace_back(" 200k ", 0);  // We allow the user selection of the 3 x WFM BW filters, (0) WFM-200K, (1) WFM-180K, (2) WFM-40K.
```

Comment:
To keep in sync with naming conventions, you might want to consider replacing these 4 lines with:  freqman_set_bandwidth_option(WFM_MODULATION, field_rxbw);

Reply:
Same comment as above . thanks,